src/main/java/com/lpvs/service/scan/LPVSDetectService.java

```diff
@@ -105,52 +111,118 @@ public LPVSDetectService(
      */
     @EventListener(ApplicationReadyEvent.class)
     public void runSingleScan() {
-        if (trigger != null && !HtmlUtils.htmlEscape(trigger).isEmpty()) {
-            log.info("Triggered single scan operation");
+        // generateReport indicates that a report should be generated (HTML or command line output)
+        boolean generateReport = false;
+        LPVSQueue webhookConfig = null;
+        List<LPVSFile> scanResult = null;
+        List<LPVSLicenseService.Conflict<String, String>> detectedConflicts = null;
+
+        // Error case when both pull request scan and local files scan are set to true
+        if (trigger != null && !trigger.isBlank() && localPath != null && !localPath.isBlank()) {
```

Comment:
As isBlank include check for a `null`, so it could be simplified to:
`if (!trigger.isBlank() && !localPath.isBlank())`